deployment/copilot/README.md

```diff
@@ -71,7 +75,7 @@ The provided CDK will output export commands once deployed that can be ran on a
 ```
 export MIGRATION_DOMAIN_SG_ID=sg-123;
 export MIGRATION_DOMAIN_ENDPOINT=vpc-aos-domain-123.us-east-1.es.amazonaws.com;
-export MIGRATION_DOMAIN_USER_AND_SECRET_ARN=admin arn:aws:secretsmanager:us-east-1:123456789123:secret:demo-user-secret-123abc
```

Comment:
Will add back in. We should evaluate in the future because white space without being wrapped in quotes can be problematic. I also wonder why the username is not in the secret.